scripts/rook.sh

```diff
@@ -115,11 +150,19 @@ deploy)
 teardown)
 	teardown_rook
 	;;
+create-block-pool)
+	create_block_pool
+	;;
+delete-block-pool)
+	delete_block_pool
+	;;
 *)
 	echo " $0 [command]
 Available Commands:
-  deploy         Deploy a rook
-  teardown       Teardown a rook
+  deploy             Deploy a rook
+  teardown           Teardown a rook
+  create-block-pool  Create a rook block pool
+  delete-block-pool  Delete a rook block pool
```

Comment:
are we using delete-block-pool any where?

Reply:
no, provided for completeness.